commands/tiering.py

```diff
@@ -525,55 +526,61 @@ async def sroom(self,
                                   required=False)):
         currentname = ctx.channel.name
         namesuffix = ""
-        if re.search(r'^[A-Za-z]\d-', currentname):
-            nameprefix = re.match(r"^[A-Za-z]\d-", currentname).group(0)
-        else:
+        
+        log.info(f'Running room in {ctx.guild.name}')
+        if not self.verify_room(currentname):
             log.warning('Error: Invalid Channel')
-            await ctx.interaction.response.send_message(embed=gen_embed(
-                title='Invalid Channel',
-                content=f'This is not a valid tiering channel. Please match the format g#-xxxxx to use this command.'),
-                ephemeral=True)
+            await ctx.reply(embed=gen_embed(title='Invalid Channel',
```

Comment:
ctx.respond should be used in these instances, as we are dealing with a slash command and need to provide a followup response to our deferred response.